osmosis-1/osmosis-1.assetlist.json

```diff
@@ -16285,6 +16285,44 @@
       "keywords": [
         "osmosis-main"
       ]
+    },
+    {
```

Comment:
Please don't edit the assetlist.json

Reply:
Sir, do I need to remove the changes from this file?